packages/infra/lib/ihe-gateway-v2-stack.ts

```diff
@@ -82,7 +82,6 @@ export class IHEGatewayV2LambdasNestedStack extends NestedStack {
           { name: "patientid", type: "string" },
           { name: "stage", type: "string" },
           { name: "requestid", type: "string" },
-          { name: "gatewayoid", type: "string" },
```

Comment:
would we need to also modify the logic in `createHivePartitionFilePath`?

Reply:
Not needed -- path parameters after our partition keys are ignored anyway.